### ISSUE TITLE
Breadcrumbs don't accept undefined as children

### ISSUE THREAD

**Issue by ediblecode** [nice-digital/nice-design-system#110]
If you want to conditionally render breadcrumbs e.g:

```js
{parentChapter && (
					<Breadcrumb
						to={`/topics/${topic.slug}/${parentChapter.slug}/`}
						elementType={Link}
					>
						{parentChapter.fullItemName}
					</Breadcrumb>
				)}
```

Then you get the typescript error:

> Type '{ children: (Element | undefined)[]; }' is not assignable to type 'BreadcrumbsProps'. Types of property 'children' are incompatible.

And you get a runtime error of: " Cannot read property 'props' of null
    at getBreadcrumbJsonLdItem" because the json ld generation loops over children to get their props.

**Comment by ediblecode** [comment]
You can get round this by doing:

```js
{parentChapter ? (
					<Breadcrumb
						to={`${topicPath}${parentChapter.slug}/`}
						elementType={Link}
					>
						{parentChapter.fullItemName}
					</Breadcrumb>
				) : (
					<></>
				)}
```

But it would be nicer to use undefined rather than a fragment

**Comment by ediblecode** [comment]
Closing - this was merged in #168
